### ISSUE TITLE
[Auto–Loot error] Lockbox contents disappearing

### ISSUE THREAD

**Issue by cachocarne** [chromiecraft/chromiecraft#2332]
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [ ] Horde

### Content Phase:

- [X] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59

### Current Behaviour

When opening a lockbox (Mithril, Thorium, etc) with auto-loot set off, contents disappear. Of course, I have plenty of space on bags and items are not "lost among my stuff" like people suggest.
Items simply vanish, they don't go to the bags or the mailbox or anything like that.

### Expected Blizzlike Behaviour

I've heard people having issues with opened lockboxes as in them behaving oddly when trying to sell them or trade them. I think something's been messed up and now the contents disappear completely. I only have this sometimes, it's not happening 100% of the time. MAYBE it is triggered after I send the open lockbox to an alt and then receive it back. Need to test this again.

I will try to make a video to prove it, but would be great if somebody could test it on PTR 

### Steps to reproduce the problem

Get a Rogue with Lockpicking skill
Set Auto–Loot OFF
Pick the lock on a Chest / Lockbox
Open box to see contents:
- If content is not touched, it will remain inside the unlocked box
- if content is looted, it will disappear instead of going into our bags

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/03c84db04dd7bce2a8aa5e527aeec2640c6fa7eb

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)
- [lua-zone-debuff](https://github.com/55Honey/Acore_ZoneDebuff)

### Customizations

None

### Server

ChromieCraft

**Comment by Shineslippers** [comment]
That's absolutely not clear. I did some tests and everything was ok.
1. `Pick Lock` - open
2. `Pick Lock` via trade window
3. Relog after `Pick Lock` or trade
4. Trade of opened boxes

Need more info.

**Comment by Azcobu** [comment]
Just had a go at ths myself with equal lack of success. Spawned 20 mithril lockboxes, sent half the boxes to an alt and back, and unlocked and opened them with auto-loot off. Looting worked fine for all of them, and contents were consistent with what the alt saw in each box, and were not lost on re-opening. There's clearly a missing factor here.

**Comment by cachocarne** [comment]
Ok, seems there must have been some bugs happening during last patch update. I experienced bugs on some quests that I had half-way completed (37/50 items but could not get more, npc showed yellow "?" but was imposible to turn-in... only solved by abandoning and starting again) and probably on the boxes that were in my inventory (quite a few) unlocked but not looted. I tried to replicate error again but there was no problem anymore, all was ok this time. I would say it was due to some obscure stuff happening during update and this issue could get closed. Thanks a lot everybody for your time and efforts.
